Deploy-CAM.ps1

```diff
@@ -116,7 +116,9 @@ Param(
     [parameter(Mandatory=$false)]
     $camManagementUserGroup = $null,
 
-    [switch]$updateSPCredential
+    [switch]$updateSPCredential,
+    [bool]$isRetrieveAgentState = $true,
```

Comment:
Maybe rename these brokerRetreiveAgentState? Name's kind of vague for the install script